src/Seq.App.EmailPlus/HandlebarsHelpers.cs

```diff
@@ -65,5 +66,58 @@ static object FromDynamic(object o)
 
             return o;
         }
+
+        static void SubstringHelper(TextWriter output, object context, object[] arguments)
+        {
+            //{{ trimString value "0" "30" }}
+            var value = arguments.FirstOrDefault();
+
+            if (value == null)
+            {
+                output.WriteSafeString("null");
```

Comment:
I know the 'pretty printing' helper substitutes `null` for `"null"`, but I'm not sure we should carry this through to substring; can we consider substrings of null to just be empty strings? What do you think?

Reply:
You're quite right; will update to output a empty string.